index.html

```diff
@@ -10,7 +10,7 @@
 <body>
 	<header>
 	</header>
-	
+
```

Comment:
Also, please install a proper linter next time. I know you work with prehistorical tools but this is not an excuse.